### ISSUE TITLE
Validate without `load_static_elements`

### ISSUE THREAD

**Issue by kazimir-malevich** [tag1consulting/goose-eggs#47]
I think we need a `validate` function/associate_function/method that doesn't include `load_static_elements`. But we have a `validate` entity and still keep `validate_and_load_static_assets`.

**Comment by jeremyandrews** [comment]
I agree that there needs to be an option to validate alone -- and `validate_and_load_static_assets()` should invoke the `validate()` function so we're not duplicating logic.

**Comment by kazimir-malevich** [comment]
Then that would be fairly simple for me to do shouldn't it?

**Comment by kazimir-malevich** [comment]
@jeremyandrews The only way I can think about doing this is by extracting the commented blocks in `load_static_elements` into private functions.

**Comment by jeremyandrews** [comment]
@kazimir-malevich please see #49 -- this is what I had in mind. Sorry for not being more clear.

**Comment by kazimir-malevich** [comment]
@jeremyandrews Thanks for the heads up. Function tested and passed.
